packages/shared/src/hooks/useTagAndSource.ts

```diff
@@ -206,7 +210,7 @@ export default function useTagAndSource({
         return { successful: false };
       }
       logEvent({
-        event_name: LogEvent.UnblockSource,
+        event_name: 'follow',
```

Comment:
let's fix that to unblock
```suggestion
        event_name: 'unblock',
```